packages/sanity/src/desk/panes/document/timeline/timelineMenu.tsx

```diff
@@ -136,10 +138,12 @@ export function TimelineMenu({chunk, mode, placement}: TimelineMenuProps) {
   const timeLabel = useFormattedTimestamp(chunk?.endTimestamp || '')
 
   const revLabel = chunk
-    ? `${upperFirst(formatTimelineEventLabel(chunk.type))}: ${timeLabel}`
-    : 'Latest version'
+    ? `${upperFirst(t(`desk.timeline.${chunk.type}`))}: ${timeLabel}`
```

Comment:
This doesn't need/shouldn't have the `upperFirst` call anymore, since we can now localize it specifically for the exact location it is used. It is also unclear whether or not all languages have the same casing rules.